java/configuration/src/main/java/sleeper/configuration/table/index/DynamoDBTableIdFormat.java

```diff
@@ -31,13 +33,21 @@ private DynamoDBTableIdFormat() {
 
     static final String TABLE_NAME_FIELD = "TableName";
     static final String TABLE_ID_FIELD = "TableId";
+    static final String ONLINE_FIELD = "Online";
+
 
     public static Map<String, AttributeValue> getItem(TableIdentity id) {
         return Map.of(
                 TABLE_ID_FIELD, createStringAttribute(id.getTableUniqueId()),
                 TABLE_NAME_FIELD, createStringAttribute(id.getTableName()));
     }
 
+    public static Map<String, AttributeValue> online(TableIdentity id) {
+        Map<String, AttributeValue> item = new HashMap<>(getItem(id));
+        item.put(ONLINE_FIELD, createBooleanAttribute(true));
+        return item;
+    }
```

Comment:
The getItem method still returns an item without an Online field at all. This is used in the update method, so updating a table identity will discard the online field completely. It should always be set explicitly and retained.

Maybe it should just be in the TableIdentity object, so we don't need a takeOffline/putOnline on the table index, just update?